integration_test.go

```diff
@@ -0,0 +1,358 @@
+package ion
```

Comment:
nit: Reminder to add licensing when we merge to official repo.

Reply:
These imports are all from the same library (Go SDK)